app/(main)/layout.tsx

```diff
@@ -38,7 +38,7 @@ export default function MainPageLayout({
         </HeaderNav>
         <HeaderNav
           label="Site controls"
-          className="fixed right-0 md:bottom-0 md:left-0"
```

Comment:
Without unsetting the definition for right at md size and up, this element stretches across the width of the screen.